app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
First of all you should split and check that length is exactly 3, then you could upack it into variables